tools/c7n_azure/tests/templates/vmscaleset.json

```diff
@@ -0,0 +1,263 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "IotHubs_cctest_iothub_name": {
```

Comment:
This template includes more things than VMSS

Reply:
I'll get this one and the other template cleaned up